jans-core/saml/src/main/java/io/jans/saml/metadata/model/LocalizedText.java

```diff
@@ -0,0 +1,33 @@
+package io.jans.saml.metadata.model;
+
+public class LocalizedText {
+
+    private String language;
+    private String text;
+    
+    public LocalizedText() {
+
+        this.language = language;
+        this.text = text;
```

Comment:
Is this code auto-generated? Otherwise it's not clear what code above means. Empty constructor with assigning variables to itself.

Also when I look on all these boilderplate builders from "io.jans.saml.metadata.builder" package I think about lombok. If we plan to heavily rely on builders maybe it's good time to add lombok to not waste time on builders in future.

Reply:
@yuriyz this has been fixed. Could you please accept the changes ?